packages/headless/src/features/search/search-actions.ts

```diff
@@ -271,51 +271,19 @@ const shouldReExecuteTheQueryWithCorrections = (
 
 export const buildSearchRequest = (state: StateNeededByExecuteSearch) => {
   const facets = getFacets(state);
+  const sharedWithFoldingRequest =
+    buildSearchAndFoldingLoadCollectionRequest(state);
```

Comment:
Suggestion: `buildBaseSearchRequest`

Reply:
I don't like `base` here, it does not mean anything. 

I would not call it base, it's really something that is explicitly shared for *only* folding load more and execute search.

I don't want maintainer in the future to tack new things in that function, or to hook up to it to actions that are different than those two, unless they understand very well what they are doing (hence the long descriptive name).